General/Traits.md

```diff
@@ -38,11 +38,13 @@ Concrete example:
 Trait named: #FamixTWithEnumValues
 	uses: {}
 	slots: {}
-	category: 'Famix-Traits-EnumValue'
+	package: 'Famix-Traits-EnumValue'
 ```
 
 > Calypso provides a menu entry to create traits. To access it, right-click on the classes list (with no class or trait selected) and select "New trait".
 
+> **Note**: In Pharo < 8, the message #named:uses:slots:package: should be replaced by #named:uses:slots:category:.  
```

Comment:
```suggestion
> **Warning**: In Pharo < 8, the message #named:uses:slots:package: should be replaced by #named:uses:slots:category:.  
```

I think we should specify that this remark is a warning. Maybe when current Pharo version will be 10 it can be optional to read. But right now it is too early, too many people still use Pharo version < 8.